packages/coil-extension/src/background/services/TabStates.ts

```diff
@@ -58,16 +58,24 @@ export class TabStates {
     const state: TabState = {
       playState: 'playing',
       stickyState: 'auto',
+      disabling: {
+        disableDomain: false,
+        disablePaymentPointer: false,
+        disableUrl: false
+      },
       frameStates: {
         [0]: this.makeFrameStateDefault()
       }
     }
     return state
   }
 
-  private makeFrameStateDefault() {
+  private makeFrameStateDefault(): FrameState {
     return {
       monetized: false,
+      monetizationDetails: null,
+      disabled: false,
+      interactive: false,
```

Comment:
the pauseWebMonetization script from the background on idle, on visibilityState === 'hidden', would set this to false